_pytest/skipping.py

```diff
@@ -121,11 +121,9 @@ def _istrue(self):
                 # "holder" might be a MarkInfo or a MarkDecorator; only
                 # MarkInfo keeps track of all parameters it received in an
                 # _arglist attribute
-                if hasattr(self.holder, '_arglist'):
-                    arglist = self.holder._arglist
-                else:
-                    arglist = [(self.holder.args, self.holder.kwargs)]
-                for args, kwargs in arglist:
+                marks = getattr(self.holder, '_marks', None) \
```

Comment:
@flub when i did this the first time (the pr we rejected at the sprint)

i did move this code into the marks module

i plan to fix it eventually, but that was not in the scope of this pr